doc/api/v1.yaml

```diff
@@ -46,23 +46,19 @@ paths:
           description: One of messages is malformed
 
   /api/v1/fragments/statuses:
-    post:
+    get:
       description: Get statuses of fragments
       operationId: FragmentStatuses
       tags:
         - fragment
-      requestBody:
-        description: List of fragment IDs
-        required: true
-        content:
-          application/json:
-            schema:
-              type: array
-              items:
-                type: string
-                pattern: '[0-9a-f]+'
-            example: ["68dcc12fe0dfe5e7b66ca6f8c959f9aa43b273e120a77fc3e4e2f04f1ecd7968"]
```

Comment:
Would be good to provide an updated example.

Reply:
Done.